gpt_computer_agent/utils/chat_history.py

```
@@ -0,0 +1,220 @@
+
+
+
+
+
+
+
+from langchain_core.messages import HumanMessage, SystemMessage, AIMessage
+
+
+try:
+    from .db import *
```

Ruff (0.8.2)

`from .db import *` used; unable to detect undefined names

(F403)